styles.css

```diff
@@ -0,0 +1,138 @@
+.container {
+	background-image: url("https://backgrounddownload.com/wp-content/uploads/2018/09/background-banner-motor.jpg");
+	background-color:#000000;
+	text-align:center;
+	background-size: contain;
+	background-repeat: no-repeat;
+	overflow:auto;
+}
+
+.banner {
+	margin-left:170px;
+	float:left;
+	color:white;
+}
+
+h1 {
+    font-size:50px;
+    margin:0px; 
+}
+
+h2 {
+	font-size:20px;
+	margin:0px;
+}
+
+ 
+ .buttonContainer{
+	width:500px;
+	overflow:auto;
+	float:right;
+	text-align:center;
+	color:white;
+}
+
+.buttonContainer div {
+	float:right;
+	width:30%;
+	height:60px;
+	background-color:red;
+	padding-top:30px;
+	border:2px solid white;
+	background-color:transparent;
+	font-size: 20px;
+	border-radius: 12px;
+}
+
+@media only screen and (max-width:680px) {
+.buttonContainer div a {
+	width:500px;
+    text-decoration: none;}
+}
+
+.buttonContainer div a{
+	padding:30px;
+	text-decoration:none;
+	color:white;
+}
+
+.biography {
+	background-image: url("https://d2v9y0dukr6mq2.cloudfront.net/video/thumbnail/mCz7hqj/videoblocks-cogs-motion-background-loop-black-red_ruvlw8acg_thumbnail-full01.png");
+	background-size:cover;
+	background-color:white;
+    background-position: center;
+	height:700px;
+	background-repeat: no-repeat;
+	overflow:auto;
+}
+
+@media only screen and (max-width:1180px) {
+	.biography .photo {
+		margin:0 auto;
+		float:none;}
+}
+
+
+.photo {
+	background-image: url("http://pcm.um.edu.my/wp-content/uploads/2017/11/empty-avatar-700x480.png");
+	height:500px;
+	width:500px;
+	color:white;
+	background-size:cover,contain;
+	background-repeat: no-repeat;
+	float:right;
+	margin-right:0px;}
+
+.about {
+	color:white;
+	float-text:left;
+	margin-left:10px;
+	padding:15px;
+	color:white;
+	text-align:center;}
+	
+@media only screen and (max-width:1180px) {
```

Comment:
this is the same media query as the one before just put all the css in the first one